lib/screen/slide.dart

```diff
@@ -0,0 +1,7 @@
+class Slide {
+  String image;
+  String heading;
+  String content;
```

Comment:
Make a new folder for model classes and then move this Slide class under that folder.